stable/wordpress/templates/deployment.yaml

```diff
@@ -28,11 +28,7 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: ALLOW_EMPTY_PASSWORD
-        {{- if .Values.allowEmptyPassword }}
-          value: "yes"
-        {{- else }}
-          value: "no"
-        {{- end }}
+          value: {{ .Values.allowEmptyPassword | quote }}
```

Comment:
Great! Can you remove the quotes from `values.yaml` too in the code block below? It's unnecessary since you're quoting it here.

```
## Set to `yes` to allow the container to be started with blank passwords
## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
allowEmptyPassword: "yes"
```